var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -69,6 +72,9 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("+cuda", when="+rocm")
     conflicts("+openmp", when="+oneapi")
 
+    conflicts("+sde", when="@:1.6.0")
+    conflicts("+sde", when="@master")
```

Comment:
Hmm, I don't get it. Are you adding a variant that always conflicts?

Reply:
Oh, this works only on `develop` Can you make it clean and enable the variant only `when=@develop` and remove the conflicts?